cumulus/ansible/tasks/inventory.py

```diff
@@ -16,7 +18,7 @@ class AnsibleInventoryHost(object):
     '''
     def __init__(self, host, **kwargs):
         self.host = host
-        self.variables = kwargs
+        self.variables = OrderedDict(sorted(kwargs.items()))
 
```

Comment:
@kotfic PTAL at the use or OrderedDict and sorted, to get consistent iteration order.